partiql-conformance-tests/src/bin/generate_comparison_report.rs

```diff
@@ -135,7 +135,7 @@ Number failing in main but now pass: {}
         ).expect("write passing_orig_failing_new heading");
         for test_name in &passing_orig_failing_new {
             comparison_report_file
-                .write_all(format!("- {}\n", test_name).as_bytes())
+                .write_all(format!("- {test_name}\n").as_bytes())
```

Comment:
These changes and from `generate_cts_report.rs` were other clippy warnings not found fixed in #288.